azure-functions-maven-plugin/README.md

```diff
@@ -109,10 +109,7 @@ Property | Required | Description
 `<deploymentType>` | false | Specifies the deployment approach you want to use.<br>The default value is **`zip`**.
 `<httpProxyHost>` | false | Specifies an optional HTTP proxy to connect to Azure through.
 `<httpProxyPort>` | false | Specifies an optional HTTP proxy port to connect to Azure through <br>The default value is **80**.
-`<localDebugAddress>` | false | Transport address for the connection. If server=n, attempt to attach to debugger application at this address. If server=y, listen for a connection at this address. The default value is **5005**.
-`<localDebugServer>` | false | If "y", listen for a debugger application to attach; otherwise, attach to the debugger application at the specified address. The default value is **"y"**.
-`<localDebugSuspend>` | false | If "y", VMStartEvent has a suspendPolicy of SUSPEND_ALL. If "n", VMStartEvent has a suspendPolicy of SUSPEND_NONE. The default value is **"n"**.
-`<localDebugTransport>` | false | Name of the transport to use in connecting to debugger application. It could be **dt_socket** or **dt_shmem** *(Windows Only)*. The default value is **"dt_socket"**. 
+`<localDebugConfig>` | false | The config string of debug options, you may visit [here](https://docs.oracle.com/javase/7/docs/technotes/guides/jpda/conninv.html) for more infos. The default value is `transport=dt_socket,server=y,suspend=n,address=5005`;
```

Comment:
info -> information

We should avoid using abbreviations in all of the documents